src/darwin/Framework/CHIP/templates/CHIPCallbackBridge-src.zapt

```diff
@@ -42,6 +42,11 @@
 {{#chip_server_cluster_attributes}}
 {{#if isArray}}
 {{#>CHIPCallbackBridge ns=parent.name                                }}{{asUpperCamelCase ../../name}}{{asUpperCamelCase ../name}}ListAttributeCallback{{/CHIPCallbackBridge}}
+{{else}}
+  {{#if_is_struct type}}
+  {{#>CHIPCallbackBridge ns=parent.name                              }}{{asUpperCamelCase ../../name}}{{asUpperCamelCase ../name}}StructAttributeCallback{{/CHIPCallbackBridge}}
+{{else}}
```

Comment:
I don't know zapt syntax but this looks wrong? Is the `{{else}}` needed before ending the `if_is_struct` check on line 49?

Reply:
Good catch.  The `{{else}}` here is not needed (though it being there is just a no-op, since the body is empty, but it's a confusing no-op).